examples/pip_parse/requirements_lock.txt

```diff
@@ -2,15 +2,26 @@
 # This file is autogenerated by pip-compile
 # To update, run:
 #
-#    pip-compile --output-file=requirements_lock.txt requirements.txt
+#    pip-compile --generate-hashes --output-file=requirements_lock.txt requirements.txt
```

Comment:
I am surprised there isn't a second place in this example where the generate-hashes argument was supplied.
If the point of the PR is to allow users to give pip flags, shouldn't there be a source file where they do so? (as opposed to this generated file?)

Reply:
I would think of this file, which happens to be generated, as the only input from the user. The fact that there is also a requirements.txt in this directory is because we are using pip-compile to create the lock file. I can modify this file to add some pip arguments for the sake of demonstration.

Reply:
Adding this turned up a small bug, so yay.